Celeste.Mod.mm/Patches/Settings.cs

```diff
@@ -157,5 +159,45 @@ class patch_Settings : Settings {
 
         #endregion
 
+        [MonoModIgnore]
+        public static extern void orig_Initialize();
+        public new static void Initialize() {
+            orig_Initialize();
+            // Load Mouse Button Bindings, which are applied to the already-loaded settings.
+            if (UserIO.Open(UserIO.Mode.Read)) {
+                (UserIO.Load<VanillaMouseBindings>("modsettings-Everest_MouseBindings") ?? new VanillaMouseBindings().Init()).Apply();
+                UserIO.Close();
+            }
+        }
+
+        public void ClearMouseControls() {
```

Comment:
I feel like this would be a better fit as a method in `VanillaMouseBindings` that you call from here.

Reply:
This matches the vanilla `Settings.SetDefault{Keyboard/Button}Controls`, and I would prefer to keep the scope for `VanillaMouseBindings` restricted to a serialization helper.